sql/moz-fx-data-shared-prod/telemetry/deanonymized_events/view.sql

```diff
@@ -0,0 +1,47 @@
+CREATE OR REPLACE VIEW
+  telemetry.deanonymized_events
```

Comment:
Are we free to use the `tmp` dataset for these kinds of views? I don't actually want this to be available to end users.

Reply:
We have one or two cases where we put views in `_derived` because they weren't intended for end users, which is what I'd recommend here.